cmake/SetupCompilerOptions.cmake

```diff
@@ -111,6 +111,20 @@ if(BLT_DEFINES)
     message(STATUS "Added \"${BLT_DEFINES}\" to definitions")
 endif()
 
+if(COMPILER_FAMILY_IS_MSVC)
+    add_definitions(-D_USE_MATH_DEFINES)
+
+    # Mark 32 bit executables large address aware so they can use > 2GB address space
```

Comment:
I had these from a previous project.  They are for 32-bit systems, so might be outdated.
(But my copy of Visual Studio still generates 32-bit executables).